### ISSUE TITLE
problem when changing directory

### ISSUE THREAD

**Issue by nielsdevleeschouwer** [fluves/pywaterinfo#32]
* pywaterinfo version: d6261dd
* Python version: python 3.7.7
* Operating System: Linux

### Description

When I change the directory I'm no longer able to request data. This has probably something to do with the location of the cache database.

### What I Did

```
from pywaterinfo import Waterinfo
import os
vmm = Waterinfo("vmm")
os.chdir('..')
df1 = vmm.get_timeseries_values(78124042, period="P1D")

--> sqlite3.OperationalError: no such table: responses
```

**Comment by stijnvanhoey** [comment]
@nielsdevleeschouwer thanks for the issue and indeed this is due to the change in working directory by which the relative load of the sqlite based cache is no longer found.

However, I'm curious why you need to change the working directory in the middle of an analysis. I do not know if we really need to support this  - for my understanding - 'edge case'? If this is required for important use cases or others facing similar issues, we can have a look at it.

**Comment by nielsdevleeschouwer** [comment]
The issue has to do with code where a change in directory happens between initialisation of the waterinfo class and the actual use of it to request data. But this is probably not best practice.

Saving the cache data in a temp folder however, can be convenient. That way all cache data is centralized and can be requested from different projects. Temp folders are also automatically cleared from time to time so that the cache data does not build up over time.

**Comment by stijnvanhoey** [comment]
@nielsdevleeschouwer see https://github.com/fluves/pywaterinfo/pull/35 for the PR. I rerun your example above and the issue is no longer there when using `os.chdir`

**Comment by stijnvanhoey** [comment]
Fixed by #35
